### ISSUE TITLE
Missing apk file in latest releases

### ISSUE THREAD

**Issue by j0hann3s** [billthefarmer/editor#208]
I do not know if this is by design, but the latest releases do not contain the apks anymore for the release. This means that my updates tracking app (Obtainium) can not update this app anylonger. I would love to be able to update without needing to install Android Developer tools and compile the app :)

**Comment by billthefarmer** [comment]
See #198, read to end. What's wrong with F-Droid?  Please check closed issues before raising a new one.

**Comment by j0hann3s** [comment]
Oh Sorry! The only issue that I checked was #190 where you mention Fdroid as alternative which was back in January but the latest release that still had an apk file was in May so I thought you continued with the files. The title "We Need Your Help" did not really seem relevant to me in my search.

I do like the idea of only one apk existing from Fdroid with their app auditing. I did not understand until now that you upload the Fdroid apk afterwards after having released the code on GitHub. I should have checked the dates of version releases and the added apk files better.

To answer the question what I think is wrong with Fdroid: I am not a fan of Fdroid using their key to sign apps just because of the security implications if the key gets stolen than half my phones apps are at risk, but thats out of the scope for this issue/question. I also love getting updates faster than what Fdroid provides in terms of delay. But I am fully content just with the new knowledge that you upload the fdroid apks here on the repo after the release :)
